eng/test-job.yml

```diff
@@ -35,29 +36,69 @@ jobs:
       osSubgroup: ${{ parameters.managedTestBuildOsSubgroup }}
       container: ${{ parameters.container }}
       testGroup: ${{ parameters.testGroup }}
-      readyToRun: ${{ parameters.readyToRun }}
       crossrootfsDir: ${{ parameters.crossrootfsDir }}
       corefxTests: ${{ parameters.coreFxTests }}
       displayNameArgs: ${{ parameters.displayNameArgs }}
       condition: ${{ parameters.condition }}
       stagedBuild: ${{ parameters.stagedBuild }}
 
-- template: run-test-job.yml
-  parameters:
-    buildConfig: ${{ parameters.buildConfig }}
-    archType: ${{ parameters.archType }}
-    osGroup: ${{ parameters.osGroup }}
-    osSubgroup: ${{ parameters.osSubgroup }}
-    ${{ if ne(parameters.managedTestBuildOsGroup, '') }}:
+      ${{ if eq(parameters.testGroup, 'innerloop') }}:
+        priBuildNamePart: 'p0_'
+        priDisplayNamePart: 'Pri0 '
+      ${{ if ne(parameters.testGroup, 'innerloop') }}:
+        priBuildNamePart: 'p1_'
+        priDisplayNamePart: 'Pri1 '
+
+- ${{ if or(eq(parameters.r2rPlatforms, ''), containsValue(parameters.jitPlatforms, 'all'), containsValue(parameters.jitPlatforms, format('{0}{1}_{2}', parameters.osGroup, parameters.osSubgroup, parameters.archType))) }}:
```

Comment:
> `containsValue(parameters.jitPlatforms, 'all')`

I think this should be `eq()` instead. I would prefer these to be as precise as possible.

Reply:
Same for the template below.

Reply:
Good point, fixed in 2nd commit.